modules/identity-provider-oidc-CR.adoc

```diff
@@ -34,21 +32,16 @@ spec:
         - name
         email:
         - email
+        groups:
+        - groups
       issuer: https://www.idp-issuer.com <6>
 ----
-<1> This provider name is prefixed to the value of the identity claim to form an
-identity name. It is also used to build the redirect URL.
+<1> This provider name is prefixed to the value of the identity claim to form an identity name. It is also used to build the redirect URL.
 <2> Controls how mappings are established between this provider's identities and `User` objects.
-<3> The client ID of a client registered with the OpenID provider. The
-client must be allowed to redirect to
-`\https://oauth-openshift.apps.<cluster_name>.<cluster_domain>/oauth2callback/<idp_provider_name>`.
-<4> Reference to an {product-title} `Secret` object containing the client secret.
-<5> List of claims to use as the identity. First non-empty claim is used. At
-least one claim is required. If none of the listed claims have a value,
-authentication fails. For example, this uses the value of the `sub` claim in the returned `id_token` as the user's identity.
```

Comment:
@stlaz I removed this sentence because I don't believe it to be true: "For example, this uses the value of the sub claim in the returned id_token as the user's identity."

I checked the 3.11 version of this file [1] and do see an id/sub config, so I'm thinking it was just leftover from that file. Let me know if this removal is correct

[1] https://docs.openshift.com/container-platform/3.11/install_config/configuring_authentication.html#OpenID

Reply:
Unless you think it's important, I'll probably just leave it off. Partly because I'm not sure exactly what to say that this example does (so unless you have a suggestion what to say for it!)